modules/UI/toolbars/ToolbarToggler.js

```diff
@@ -35,7 +35,6 @@ function hideToolbar(force) { // eslint-disable-line no-unused-vars
     toolbarTimeoutObject = null;
 
     if (Toolbar.isHovered()
-            || APP.UI.isRingOverlayVisible()
```

Comment:
Fixed in https://github.com/jitsi/jitsi-meet/commit/ecfc56461ed397206a9808e6d980b1e26e76fb44